src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -34,6 +34,7 @@ public class LoginSsoPageViewModel : BaseViewModel
         private readonly ICryptoService _cryptoService;
 
         private string _orgIdentifier;
+        private bool _useEphemeralWebBrowserSession = false;
```

Comment:
⛏️ bool is `false` by default.
```suggestion
        private bool _useEphemeralWebBrowserSession;
```